front-end/src/pages/Home/components/Search/Search.jsx

```diff
@@ -0,0 +1,43 @@
+// Search.js
+import React, { useState, useEffect } from 'react';
+import './Search.scss';
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+
+const Search = ({ onSearch ,setIsSearching}) => {
```

Comment:
Again, I believe setIsSearching is not needed, we can depend on the searchQuery inside useNotes.